x-pack/plugins/security_solution/cypress/integration/alerts.spec.ts

```diff
@@ -24,37 +25,35 @@ import {
   waitForAlertsToBeLoaded,
   markInProgressFirstAlert,
   goToInProgressAlerts,
+  waitForAlertsIndexToBeCreated,
 } from '../tasks/alerts';
-import { removeSignalsIndex } from '../tasks/api_calls/rules';
+import { createCustomRuleActivated } from '../tasks/api_calls/rules';
 import { cleanKibana } from '../tasks/common';
-import { esArchiverLoad, esArchiverUnload } from '../tasks/es_archiver';
+import { waitForAlertsToPopulate } from '../tasks/create_new_rule';
 import { loginAndWaitForPage } from '../tasks/login';
+import { refreshPage } from '../tasks/security_header';
 
 import { DETECTIONS_URL } from '../urls/navigation';
 
 describe('Alerts', () => {
   context('Closing alerts', () => {
     beforeEach(() => {
       cleanKibana();
-      removeSignalsIndex();
-      esArchiverLoad('alerts');
       loginAndWaitForPage(DETECTIONS_URL);
-    });
-
-    afterEach(() => {
-      esArchiverUnload('alerts');
+      waitForAlertsPanelToBeLoaded();
```

Comment:
I'm a little bit wary of moving these steps into a beforeEach because of the fail-fast behavior and potential for false negatives, but I think that the speed and DRYness of this approach make it an acceptable tradeoff.